onnx/helper.py

```diff
@@ -482,6 +486,59 @@ def make_tensor_value_info(
     return value_info_proto
 
 
+def make_sparse_tensor_value_info(
+        name,  # type: Text
+        elem_type,  # type: int
+        shape,  # type: Optional[Sequence[Union[Text, int]]]
+        doc_string="",  # type: Text
+        shape_denotation=None,  # type: Optional[List[Text]]
+):  # type: (...) -> ValueInfoProto
+    """Makes a ValueInfoProto based on the data type and shape."""
```

Comment:
nit: of type SparseTensor

Reply:
This comment is not clear to me. `Data type` refers to the `elem_type` and the return type is `ValueProtoInfo`.

Reply:
I meant something like this: Makes a ValueInfoProto of type Sparse Tensor based on the elem_type and shape

Reply:
this is just to improve readability... feel free to ignore if it is confusing... but there are 2 types in play here... 1. type of typeproto and the underlying data type of the tensor proto

Reply:
And the actual return type which is ValueInfoProto